peer/doc.go

```diff
@@ -104,24 +104,20 @@ of a most-recently used algorithm.
 Message Sending Helper Functions
 
 In addition to the bare QueueMessage function previously described, the
-PushAddrMsg, PushGetBlocksMsg, PushGetHeadersMsg, and PushRejectMsg functions
-are provided as a convenience.  While it is of course possible to create and
-send these message manually via QueueMessage, these helper functions provided
-additional useful functionality that is typically desired.
+PushAddrMsg, PushGetBlocksMsg, and PushGetHeadersMsg, functions are provided as
```

Comment:
Unnecessary extra comma after `and PushGetHeadersMsg`

Reply:
Removed.